css/style.css

```diff
@@ -217,35 +226,119 @@ q::after {
     margin: 0;
     padding: 100px 50px 50px;
   }
-}
 
-/**
- ********************
- *
- *  height: auto 
-   no height, adaptable 
+  /*  ____________  store page and form   ___________________ */
 
+  .form_wrapper {
+    width: 100%;
+    display: flex;
+  }
 
-   max height 
+  .form {
+    width: 50%;
+  }
 
+  .form_poster {
+    width: 50%;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
 
+  .form_field {
+    width: 75%;
+    margin: 20% auto;
+    text-align: left;
+    border: 0;
+  }
 
-   weight: 100%
-   height: auto 
- *
- *
+  legend {
+    font-size: 38px;
+    color: var(--orange-dark);
+    font-style: normal;
+    font-weight: 500;
+    text-align: left;
+    padding-bottom: 40px;
+  }
 
- __________flex direction: column; for mobile, flex direction row for desktop
- products and product features
- feature
- feature
- feature 
+  input {
+    width: 100%;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    padding: 10px 0 5px 5px;
+    border-color: var(--grey-light);
+    border-radius: 4px;
+  }
+  .name_wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+  .first_name_wrapper {
+    width: 45%;
+  }
 
+  .last_name_wrapper {
+    width: 50%;
+  }
 
- section - big blok
+  .city_wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 30px;
+  }
 
-utility classes - small class
+  .city {
+    width: 60%;
+    display: flex;
+    flex-direction: column;
+  }
 
-> direct child 
+  .city_options {
+    margin-top: 10px;
+    margin-bottom: 20px;
+    padding: 10px 10px 5px 5px;
+    border-color: var(--grey-light);
+    border-radius: 4px;
+  }
 
-*/
+  .postcode {
+    width: 35%;
+  }
+  .radio_title {
+    font-weight: 400;
+  }
+  .radio_wrapper {
+    width: 80%;
+    display: flex;
+    align-items: center;
+    margin: 0;
+  }
+  .radio {
+    width: 8%;
+    margin: 0;
+  }
+
+  .radio_label {
+    margin: 0 15px 0 0;
+  }
+
+  .checkbox_wrapper {
+    width: 100%;
+    display: inline-flex;
+    align-items: center;
+    padding: 80px 0 30px;
+  }
+  .checkbox {
+    width: 5%;
+    align-items: center;
+    margin: 0;
+  }
+  .terms {
+    color: var(--orange-dark);
+    text-decoration: underline;
+  }
+  .submit_btn {
+    width: 50%;
+  }
```

Comment:
Rather than setting a % here (which will be determined by how big the container is), maybe try size it just by setting padding on the button? This will make sure that the button doesn't get too small or too big

Reply:
I did not take that under account, thank you very much for pointing this.